rcp/org.testeditor.rcp4.views.projectexplorer/src/org/testeditor/rcp4/views/projectexplorer/TEContentProvider.xtend

```diff
@@ -62,6 +58,21 @@ class TEContentProvider implements ITreeContentProvider {
 		return null
 	}
 
+	/**
+	 * returns the parent folder as ClasspathEntry or null if it is not a ClasspathEntry.
+	 */
+	def IClasspathEntry getParentClassPathOf(IResource element) {
```

Comment:
- Is `private` visibility sufficient here?
- Method name: how about `getParentClasspathEntry`?
- `Classpath` is a single word and doesn't need camel-case.